Production_scripts/Process_extracts/Produce_extract_tests/extract_nrs_tests.R

```diff
@@ -0,0 +1,38 @@
+####################################################
+# Name of file - extract_nrs_tests.R
+# Original Authors - Bateman McBride
+# Original Date - July 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               acute processed file.
+#####################################################
+
+# Load packages
+library(createslf)
+
+# Read in Data-----------------------------------------
+
+year <- "1920"
+
+# Read new data file
+new_data <- readr::read_rds(get_source_extract_path(year, "Deaths", ext = "rds"))
```

Comment:
```suggestion
new_data <- readr::read_rds(get_source_extract_path(year, "Deaths"))
```
We do not need to specify `ext = "rds' here when using the `get_source_extract_path` function as within the function this calls on `get_file_path` which specifies rds files by default. I think some of these other tests will be similar so i'll make an issue to go back and check this.